util/reggen/gen_dv.py

```diff
@@ -42,6 +42,11 @@ def sv_base_addr(top: Top, if_name: Tuple[str, Optional[str]]) -> str:
     return "{}'h{:x}".format(top.regwidth, top.if_addrs[if_name])
 
 
+def is_top_inst(b) -> bool:
```

Comment:
I think this might be easier to understand if it was the other way around (`is_outside_top`, or something). I didn't understand what this did until I'd read the rest of the patch, but I think it might be more obvious if you switch it around.